lib/ui/util/dimens.dart

```diff
@@ -1,7 +1,19 @@
+import 'package:flutter/material.dart';
```

Comment:
For info, this class exists in aedappfm

Reply:
https://github.com/archethic-foundation/archethic-dapp-framework-flutter/issues/34